_posts/en/newsletters/2024-02-21-newsletter.md

```diff
@@ -0,0 +1,309 @@
+    they simply didn’t get a match, they can ask for the raw data to be
+    shared to investigate further."
+
+    If the process is eventually found acceptable (perhaps with
+    additional refinements), it's possible future versions of Bitcoin
+    Core could be shipped with ASMaps and the feature enabled by
+    default, improving resistance to eclipse attacks.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Multiparty coordination protocol NWC announced:**
+  [Nostr Wallet Connect (NWC)][nwc blog] is a coordination protocol to
+  facilitate communications in interactive use cases involving multiple parties.
+  While the initial focus of NWC is Lightning, interactive protocols
+  like [joinpools][topic joinpools], [Ark][topic ark], [DLCs][topic dlc], or
+  [multisignature][topic multisignature] schemes could eventually benefit from
+  the protocol.
+
+- **Mutiny Wallet v0.5.7 released:**
+  The [Mutiny Wallet][mutiny github] release adds [payjoin][topic payjoin] support
+  and makes improvements to NWC and LSP features.
+
+- **GroupHug transaction batching service:**
+  [GroupHug][grouphug github] is a [batching][scaling payment batching] service
+  using [PSBTs][topic psbt], with [limitations][grouphug blog].
+
+- **Boltz announces taproot swaps:**
+  Non-custodial swap exchange Boltz [announced][boltz blog] an upgrade
+  to their atomic swap protocol to use [taproot][topic
+  taproot], [schnorr signatures][topic schnorr signatures], and [MuSig2][topic musig].
+
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [Bitcoin Core #27877][] updates Bitcoin Core's wallet with a new [coin
+  selection][topic coin selection] strategy, CoinGrinder (see
+  [Newsletter #283][news283 coingrinder]).  This strategy is intended to
+  be used when estimated feerates are high compared to their long-term
+  baseline, allowing the wallet to create small transactions now (with
+  the consequence that it may need to create larger transactions at a
+  later time, hopefully when feerates are lower).
+
+- [BOLTs #851][] adds support for [dual funding][topic dual funding] to
```

Comment:
```suggestion
- [BOLT #851][] adds support for [dual funding][topic dual funding] to
```
I'm not sure this suggestion is exactly correct because it's also a link, but "BOLTs" should be singular?

Reply:
Weve always used BOLTs and its also the name of the repo: https://github.com/lightning/bolts